app/serializers/users/show_serializer.rb

```diff
@@ -0,0 +1,22 @@
+module Users
+  class ShowSerializer < ActiveModel::Serializer
+    attributes :id, :first_name, :last_name, :email, :image_url,
+               :rents_counter, :comments_counter, :unread_notifications_count
+
+    def image_url
+      nil
```

Comment:
why are you creating methods whit nil values 🤔  ? if the value is empty it will be nil by default

Reply:
'cause in apiary it shows this attribute null and I didn't have this attribute in fields of model, and if I let the attribute without reference to an attribute of the model or in this case in a method in the same class the raise an error